src/screens/create-account/create-new-wallet/create-new-wallet.tsx

```diff
@@ -0,0 +1,83 @@
+import { Formik } from 'formik';
+import React, { FC } from 'react';
+import { Text, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { ButtonLargePrimary } from '../../../components/button/button-large/button-large-primary/button-large-primary';
+import { CheckboxLabel } from '../../../components/checkbox-description/checkbox-label';
+import { Divider } from '../../../components/divider/divider';
+import { HeaderBackButton } from '../../../components/header/header-back-button/header-back-button';
+import { HeaderTitle } from '../../../components/header/header-title/header-title';
+import { useNavigationSetOptions } from '../../../components/header/use-navigation-set-options.hook';
+import { InsetSubstitute } from '../../../components/inset-substitute/inset-substitute';
+import { Label } from '../../../components/label/label';
+import { NewSeedPhraseAttention } from '../../../components/new-seed-phrase-attention/new-seed-phrase-attention';
+import { ScreenContainer } from '../../../components/screen-container/screen-container';
+import { FormCheckbox } from '../../../form/form-checkbox';
+import { FormMnemonicCreate } from '../../../form/form-mnemonic-create';
+import { submitSeedPhraseAction } from '../../../store/wallet/wallet-actions';
+import { formatSize } from '../../../styles/format-size';
+import {
+  createNewWalletInitialValues,
+  createNewWalletValidationSchema,
+  CreateNewWalletFormValues
+} from './create-new-wallet.form';
+import { useCreateNewWalletStyles } from './create-new-wallet.styles';
+
+type CreateNewWalletProps = {
+  onFormSubmitted: () => void;
+};
+
+export const CreateNewWallet: FC<CreateNewWalletProps> = ({ onFormSubmitted }) => {
+  const dispatch = useDispatch();
+  const styles = useCreateNewWalletStyles();
+
+  const handleSubmit = ({ seedPhrase }: CreateNewWalletFormValues) => {
+    dispatch(submitSeedPhraseAction(seedPhrase));
```

Comment:
do not expose `seedPhrase` to the global state (it could be read from any place in the app). Store it locally (inside CreateAccount component useState and pass as prop to the next screen)